autoblocks/_impl/util.py

```diff
@@ -50,4 +50,4 @@ async def sem_coro(coro: Coroutine):
 
     # return_exceptions=True causes exceptions to be returned as values instead
     # of propagating them to the caller. this is similar in behavior to Promise.allSettled
-    await asyncio.gather(*(sem_coro(c) for c in coroutines), return_exceptions=True)
+    return await asyncio.gather(*(sem_coro(c) for c in coroutines), return_exceptions=True)
```

Comment:
Add return so I can gather evaluator events as they finish

Reply:
can you update the types on the signature

Reply:
Added type `Future[list]` - I don't think we can get more specific than that